excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SearchPartyValidatorTest.java

```diff
@@ -90,7 +93,31 @@ void shouldValidateAllFields() {
         verifyDotNotationValidatorCallMadeFor(ColumnName.SEARCH_PARTY_DOB,
                                               searchPartyEntity1.getSearchPartyDob());
         verifyDotNotationValidatorCallMadeFor(ColumnName.SEARCH_PARTY_DOD,
-            searchPartyEntity1.getSearchPartyDod());
+                                              searchPartyEntity1.getSearchPartyDod());
+
+        verifyDotNotationValidatorCallNeverMadeFor(ColumnName.SEARCH_PARTY_COLLECTION_FIELD_NAME);
+    }
+
+    @DisplayName("should validate all fields")
+    @Test
+    void shouldOnlyValidateCollectionField() {
```

Comment:
Please rename function and DisplayName e.g. "should validate all fields with collection field".

NB: RDM-13106 should fill in the gaps.

Reply:
Fixed in [3522292](https://github.com/hmcts/ccd-definition-store-api/pull/1130/commits/3522292204d6d0bc99422bd9abf6c58bbe2cbf8f)